test/module/irohad/torii/torii_transport_command_test.cpp

```diff
@@ -219,10 +220,10 @@ TEST_F(CommandServiceTransportGrpcTest, ListToriiPartialInvalid) {
 
   size_t counter = 0;
   EXPECT_CALL(*proto_tx_validator, validate(_))
-      .Times(kTimes)
+      .Times(AtLeast(1))
```

Comment:
this change seems not reasonable. only the last tx is invalid => all should be validated.

Reply:
This change is consistent with other endpoints. A message is valid only if all the transactions in the message are valid.

Reply:
then how do we make sure that all txes are checked? in this test, the first 4 txes are good, so validation is expected for them and for the next after them. this gives `kTimes`.